.eslintrc.json

```diff
@@ -1,16 +1,21 @@
 {
     "env": {
         "browser": true,
-        "webextensions": true
+        "webextensions": true,
+        "jest/globals": true
     },
     "extends": [
+        "plugin:jest/recommended",
         "plugin:import/recommended",
         "plugin:import/react",
         "plugin:promise/recommended",
         "plugin:react/recommended",
         "standard",
         "standard-react"
     ],
+    "globals": {
+        "db": true
+    },
```

Comment:
I thought so too at first. But if you refer to [line](https://github.com/WebMemex/webmemex-extension/pull/104/files#diff-cd67c91cd27eef6025852e6802ed2512R5) , it takes advantage of the jest mocking feature and uses db as a global variable. Removing it from the ```globals``` causes it to throw an error stating db is note defined.